KsApi/models/Project.Country.swift

```diff
@@ -26,7 +26,7 @@ extension Project {
     public static let hk = Country(countryCode: "HK", currencyCode: "HKD", currencySymbol: "$",  maxPledge: 70_000, minPledge: 10, trailingCode: true)
     public static let ie = Country(countryCode: "IE", currencyCode: "EUR", currencySymbol: "€",  maxPledge: 7_000, minPledge: 1, trailingCode: false)
     public static let it = Country(countryCode: "IT", currencyCode: "EUR", currencySymbol: "€",  maxPledge: 7_000, minPledge: 1, trailingCode: false)
-    public static let jp = Country(countryCode: "JA", currencyCode: "JPY", currencySymbol: "¥",  maxPledge: 1_200_000, minPledge: 100, trailingCode: false)
```

Comment:
Hard to keep up with all those ISO standards!

Reply:
Haha, it was a typo. `jp` was the static for a reason, but `ja` is the language code, so confusion!